pom.xml

```diff
@@ -2648,6 +2648,13 @@
       </modules>
     </profile>
 
+    <profile>
+      <id>kubernetes</id>
+      <modules>
+        <module>resource-managers/kubernetes/core</module>
```

Comment:
both yarn and meson don't have a sub-directory called `core`, are we going to add more modules?

Reply:
See https://github.com/apache/spark/pull/19468#discussion_r145250461